tests/codeception/acceptance/Media/MediaPermissionsCest.php

```diff
@@ -16,6 +16,17 @@ public function testAdminPerms(AcceptanceTester $I) {
     $I->canSee('Embed Code');
   }
 
+  /**
+   * Test site embedder perms
+   */
+  public function testSiteEmbedderPerms(AcceptanceTester $I) {
+    $I->logInWithRole('site_embedder');
```

Comment:
After reducing the permissions for the `site_embedder` what you can do to log in as one is:
```
$user = $I->createUserWithRoles(['site_embedder','site_manager']);
$I->logInAs($user->id());
```

Reply:
This actually needed `$I->logInAs($user->getUsername());` but same thing.

I made a pr in stanford-caravan to accept user ids there, since the method sig seemed to imply it was allowed.